gatsby-node.js

```diff
@@ -150,7 +150,9 @@ class Plugin {
 
 		try {
 			const baseUrl = new URL(url);
-			const basePath = baseUrl.pathname;
+			let basePath = baseUrl.pathname;
+
+			basePath = basePath.endsWith('/') ? basePath.slice(0, -1) : basePath;		
```

Comment:
as per @licitdev's comment:

```suggestion
			const basePath = baseUrl.pathname.endsWith('/') ? baseUrl.pathname.slice(0, -1) : baseUrl.pathname;
```